README.md

```diff
@@ -149,6 +157,18 @@ your nodes depending when the images were pulled.  Starting with the 3.7 release
 in the future.  But if you started using this image prior to 3.7, verification of your Kubernetes plugin configurations for the image pull
 policy used is warranted to guarantee consistency around what image is being used on each of your nodes.
 
+Jenkins security advisories, this image, and the `oc` binary
+---------------------------------
+
+Any updates to Jenkins core or the plugins we include in the OpenShift Jenkins master image will only occurs in the v3.11 and v4.x 
```

Comment:
s/occurs/occur/

Reply:
fix pushed

also tweaked this advisory section slightly based on my re-reading

ptal